src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -3,3 +3,6 @@ framework:
         storage_id: session.storage.mock_file
     profiler:
         enabled: false
+
+sulu_document_manager:
```

Comment:
i dont see the need currently the `document_manager` bundle is a bit different because it has a bigger impact on the system. maybe we could try to introduce a "full" new environment where preview extends the website environment?